__test__/components/CampaignInteractionStepsForm.test.js

```diff
@@ -635,14 +638,13 @@ describe("CampaignInteractionStepsForm", () => {
       beforeEach(async () => {
         await setupTest();
 
-        const startedCampaign = await createStartedCampaign();
```

Comment:
Interaction steps of campaigns that have already started are [unable to be deleted](https://github.com/MoveOnOrg/Spoke/blob/2e8c479ef7e4071bc903bdf882334488b50f2910/src/server/api/schema.js#L442-L446), so I changed the test to use a campaign that hasn't started yet.